.circleci/config.yml

```diff
@@ -26,7 +26,64 @@ e2eTest: &e2eTest
         command: ./e2e-harness teardown
         when: always
 
-version: 2
+version: 2.1
+
+orbs:
+  app-catalog:
+    executors:
+      my-executor:
+        docker:
+          - image: circleci/golang:1.12.4
+  
+    commands:
+      create-tarball:
+        steps:
+          - run:
+              name: "Creates a tarball file"
+              command: |
+                set -o errexit
+                set -o nounset
+                set -o pipefail
+      
+                readonly HELM_DIR=helm/${CIRCLE_PROJECT_REPONAME}-chart
+                readonly HELM_URL=https://storage.googleapis.com/kubernetes-helm
+                readonly HELM_TARBALL=helm-v2.11.0-linux-amd64.tar.gz
+      
+                echo_green() {
+                  echo -e "\e[32m${1}"
+                }
+                echo_red() {
+                  echo -e "\e[31m${1}"
+                }
+                echo_yellow() {
+                  echo -e "\e[33m${1}"
+                }
+
+                setup_helm_client() {
+                  echo "Setting up Helm client..."
+                  curl --user-agent curl-ci-sync -sSL -o "${HELM_TARBALL}" "${HELM_URL}/${HELM_TARBALL}"
+                  tar xzfv "${HELM_TARBALL}"
+                  PATH="$(pwd)/linux-amd64/:$PATH"
+                  helm init --client-only
+                }
+
+                echo "Setting up Helm package..."
+                setup_helm_client
+
+                sed -i -e "s/\[\[ \.SHA \]\]/${CIRCLE_SHA1}/g" $(find $HELM_DIR -type f)
```

Comment:
@kopiczko How do you feel about this?

Reply:
I don't have an issue with the command itself, but we should replace `.SHA` with `.VERSION`.

Reply:
Using version makes me feel that we will need to change all charts and we can break some backward compability.
Changing SHA for version is just a matter of naming? Or are we replacing the current system x.x.x-SHA for only VERSION which contains only the SHA? With the second one we could lose the possibility of notifying major/minor version. Is this right?

Reply:
We need to replace current `v1.0.0-[[ .SHA ]]` with `[[ .VERSION ]]`. Otherwise how we can set (git) tagged version?

Reply:
I understand now. Makes sense if we are doing that tagging strategy